contracts/dev-utils/compiler.json

```diff
@@ -7,7 +7,7 @@
         "evmVersion": "constantinople",
         "optimizer": {
             "enabled": true,
-            "runs": 10000,
+            "runs": 1666,
```

Comment:
Is this related?

Reply:
Yes, this is needed in order to reduce the size of the compiled bytecode for the DevUtils contract, to make it deployable.